src/components/portfolio/FormUploadPortfolio.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+
+const FormUploadPortfolio = ({onSubmit, onChange, multiple}) => {
+  const [isFileSelected, setIsFileSelected] = useState(false);
+
+  useEffect(() => {
```

Comment:
Do we really need this `useEffect`? without dependencies it works only once during loading the component, but we already have `useState` with default `isFileSelected` value === false